src/selenium/sessionPage.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const sessionPage = Object.create(BasePage);
+
+sessionPage.getSessionTitle = function() {
+  const self = this;
+  const titleId = 'title-1090';
+
+  const sessionPromise = new Promise(function(resolve) {
+    const title = self.find(By.id(titleId)).getText();
+
+    resolve(title);
+  });
+
+  return sessionPromise;
+};
+
+sessionPage.getSessionBackgroundColor = function() {
+  const self = this;
+  const titleId = 'title-1090';
+
+  const sessionPromise = new Promise(function(resolve) {
+    const backgroundColor = self.find(By.id(titleId)).getCssValue('background-color');
+
+    resolve(backgroundColor);
+  });
+
+  return sessionPromise;
+};
+
+sessionPage.getSpeakerName = function() {
+  const self = this;
+  const speaker = 'desc-speaker-name';
+
+  const sessionPromise = new Promise(function(resolve) {
+    const title = self.find(By.className(speaker)).getText();
+
+    resolve(title);
+  });
+
+  return sessionPromise;
+};
+
+sessionPage.jumpToTrack = function() {
+  const self = this;
+  const trackClass = 'session-ul';
+  const pageVertScrollOffset = 'return window.scrollY';
+
+  const trackPromise =  new Promise(function(resolve) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'const' statement. (one-var)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)